lms/djangoapps/learner_home/views.py

```diff
@@ -246,7 +246,7 @@ def get_course_programs(user, course_enrollments, site):
         }
     }
     """
-    meter = ProgramProgressMeter(site, user, enrollments=course_enrollments)
+    meter = ProgramProgressMeter(site, user, enrollments=course_enrollments, include_course_entitlements=True)
```

Comment:
This doesn't actually change anything because this defaults to true, but I wanted to be explicit.